LocoControl.cs

```diff
@@ -57,64 +52,5 @@ public static bool RunCommand(LocoControllerBase controller, NameValueCollection
             }
             return true;
         }
-
-        public static class ControllerUpdatePatches
-        {
-            private static bool ApproximatelyEqual(float a, float b, float epsilon = 0.01f)
-            {
-                if (a == b)
-                    return true;
-                float diff = a - b;
-                return diff >= -epsilon && diff <= epsilon;
```

Comment:
This check was to prevent data spam back to the client for tiny control changes. I would like to see it retained, because the packet spam could cause slow data connections (e.g. cellular connections to a phone) to be saturated.